floodlight/core/events.py

```diff
@@ -61,3 +95,148 @@ def custom(self):
         )
 
         return custom
+
+    @property
+    def essential_missing(self):
+        missing_columns = [
+            col for col in essential_events_columns if col not in self.essential
+        ]
+
+        if not missing_columns:
+            return None
+        else:
+            return missing_columns
+
+    @property
+    def essential_invalid(self):
+        incorrect_columns = [
```

Comment:
why not stick with invalid, i.e. ``invalid_columns`` ? Applies for all similar properties